staging/src/k8s.io/client-go/tools/clientcmd/client_config_test.go

```diff
@@ -330,6 +330,31 @@ func TestCertificateData(t *testing.T) {
 	matchByteArg(keyData, clientConfig.TLSClientConfig.KeyData, t)
 }
 
+func TestProxyURL(t *testing.T) {
+	proxyURL := "socks5://example.com"
```

Comment:
```suggestion
	proxyURL := "http://example.com"
```

Reply:
(not `https` on purpose as part of proxy are in `http` URLS, tunnelling `HTTPS` within)
could be changed to `https`

Reply:
It doesn't matter for this test. We are testing config plumbing to rest.Config here.